### DISCUSSION TITLE
Recover filer server data

### DISCUSSION THREAD

**Discussion by summerisc** [seaweedfs/seaweedfs#1854] [Q&A]
If the filer server storage db table is accidentally dropped, while other servers like volume work fine, is there a way to recover the filer information somehow? Thanks

**Comment by kmlebedev** [comment]
You need to use DB to make a backup and restore if necessary

**Comment by chrislusf** [comment]
No. But you can use `weed filer.meta.backup` to back up filer meta data into a remote filer store.
